packages/stryker-html-reporter/src/templates/sourceFile.tsx

```diff
@@ -7,7 +7,15 @@ import { ScoreResult, SourceFile, MutantResult, MutantStatus } from 'stryker-api
 import Breadcrumb from '../Breadcrumb';
 import { mutantTable } from './mutantTable';
 
+export interface NumberedMutant {
+    index: number;
```

Comment:
Do we need to save the index or can we generate the index when we loop over it?

Reply:
We don't need it indeed. But it does make it more explicit. Do you want me to remove it?

Reply:
Having an index is fine, it's just that this adds another interface.

Reply:
True. Maybe we should add mutant id to the `MutantResult` interface at a later stage, so all reporters report on the same mutants with one Id